pkg/cli/cmds/agent.go

```diff
@@ -62,6 +64,16 @@ var (
 		EnvVar:      "K3S_RESOLV_CONF",
 		Destination: &AgentConfig.ResolvConf,
 	}
+	ExtraKubeletArgs = cli.StringSliceFlag{
+		Name:  "kubelet-args",
```

Comment:
For all names can you change then to singular for example `kubelet-arg` not `kubelet-args`